xrmocap/utils/eval_utils.py

```diff
@@ -137,6 +161,27 @@ def align_convention_mask(pred_keypoints3d_raw: Keypoints,
     else:
         gt_keypoints3d = gt_keypoints3d_raw
 
+    pred_kps3d_mask = pred_keypoints3d.get_mask()
+    pred_kps3d = pred_keypoints3d.get_keypoints()[..., :3]
+    if pred_nose is not None:
+        pred_kps3d = add_campus_jaw_headtop(pred_nose, pred_kps3d)
+        pred_kps3d_mask = add_campus_jaw_headtop_mask(pred_kps3d_mask)
+
+    gt_kps3d_mask = gt_keypoints3d.get_mask()
+    gt_kps3d = gt_keypoints3d.get_keypoints()[..., :3]
+    if gt_nose is not None:
+        gt_kps3d = add_campus_jaw_headtop(gt_nose, gt_kps3d)
+        gt_kps3d_mask = add_campus_jaw_headtop_mask(gt_kps3d_mask)
+
+    pred_kps3d = np.concatenate((pred_kps3d, pred_kps3d_mask[..., np.newaxis]),
+                                axis=-1)
+    pred_keypoints3d = Keypoints(
+        kps=pred_kps3d, mask=pred_kps3d_mask, convention=eval_kps3d_convention)
+    gt_kps3d = np.concatenate((gt_kps3d, gt_kps3d_mask[..., np.newaxis]),
+                              axis=-1)
+    gt_keypoints3d = Keypoints(
+        kps=gt_kps3d, mask=gt_kps3d_mask, convention=eval_kps3d_convention)
+
     if pred_kps3d_convention != gt_kps3d_convention:
```

Comment:
For mvpose `pred_kps3d_convention` is coco, `gt_kps3d_convention` is campus and `eval_kps3d_convention` is campus, but it will set off warning